code/game/objects/items/twohanded.dm

```diff
@@ -417,9 +417,9 @@
 	var/in_mouth = ""
 	if(iscarbon(user))
 		var/mob/living/carbon/C = user
-		if(C.wear_mask == src)
+		if(C.wear_mask)
 			in_mouth = ", barely missing their nose"
-	. = "<span class='warning'>[user] swings [user.p_their()] [src][in_mouth]. [user.p_they()] light[user.p_s()] [A] in the process.</span>"
+	. = "<span class='warning'>[user] swings [user.p_their()] [name][in_mouth]. They light their [A.name] in the process.</span>"
```

Comment:
` -	. = "<span class='warning'>[user] swings [user.p_their()] [name][in_mouth]. [user.p_they(TRUE)] light[user.p_s()]  [user.p_their()] [A.name] in the process.</span>" `

Reply:
true for the first arg to `p_they` capitalizes the output.